frappe/tests/test_search.py

```diff
@@ -13,6 +13,11 @@ def test_search_field_sanitizer(self):
 		result = frappe.response['results'][0]
 		self.assertTrue('User' in result['value'])
 
+		# check if obtained result is sorted or not
+		search_link('DocType', '', query=None, filters=None, page_length=20, searchfield='name')
```

Comment:
This should be added as a separate test, perhaps as `test_search_field_order` as you want to test the order...Also, mention a key instead of `''`. At it's current state, this test doesn't do much.

Even the coverage reduced after adding this 😅 


----

btw, the sider recommendation looks good. Incorporate it too?

Reply:
If I put any key besides ' ', then I would have to use the `relevance_sorter` method for sorting and then comparing the result as the current method of sorting is just plain sorting. So would that be fine @gavindsouza ?

Reply:
yes, will incorporate that too,